### DISCUSSION TITLE
There are multiple appimage files on releases

### DISCUSSION THREAD

**Discussion by karam72** [srevinsaju/guiscrcpy#302] [Q&A]
Hi,

What is the difference between both:
- guiscrcpy-v4.11.3.dev.m.glibc2.31-x86_64.AppImage
- guiscrcpy-v4.11.3.dev.r.glibc2.31-x86_64.AppImage

on the release page:
https://github.com/srevinsaju/guiscrcpy/releases/tag/v4.11.3

Thank you

**Comment by srevinsaju** [comment]
@karam72 
* `guiscrcpy-v4.11.3.dev.m.glibc2.31-x86_64.AppImage` does not have scrcpy included in the appimage
* `guiscrcpy-v4.11.3.dev.r.glibc2.31-x86_64.AppImage` has scrcpy included.

**Reply by luni3359** [reply]
I had this question too, what do the letters mean?
